modules/nw-ovn-ipsec-enable.adoc

```diff
@@ -23,3 +23,41 @@ As a cluster administrator, you can enable IPsec encryption after cluster instal
 $ oc patch networks.operator.openshift.io cluster --type=merge \
 -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"ipsecConfig":{ }}}}}'
 ----
+
+.Verification
+
+. To find the names of the OVN-Kubernetes control plane pods, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -l app=ovnkube-master -n openshift-ovn-kubernetes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   READY   STATUS    RESTARTS   AGE
+ovnkube-master-fvtnh   6/6     Running   0          122m
+ovnkube-master-hsgmm   6/6     Running   0          122m
+ovnkube-master-qcmdc   6/6     Running   0          122m
+----
+
+. Verify that IPsec is enabled on your cluster:
++
+[source,terminal]
+----
+$ oc -n openshift-ovn-kubernetes rsh ovnkube-master-<XXXXX> \
+  ovn-nbctl --no-leader-only get nb_global . ipsec
+----
++
+--
+where:
+
```

Comment:
I'm sorta surprised this still worked w/o the continuation, but I would try adding this in here anyway for completeness.
```suggestion
+
```

Reply:
Oh sorry, nvm, I see now that this was in an open block `--`, so yeah, not necessary!